internal/services/vault_service.go

```diff
@@ -79,3 +79,12 @@ func (s *Storage) GetLeaderVaultCount() (int64, error) {
 	}
 	return count, nil
 }
+
+func (s *Storage) GetLeaderVaultTotalBalance() (int64, error) {
+	//return sum of balance of all leader vaults
+	var totalBalance int64
+	if err := s.db.Model(&models.Vault{}).Select("sum(balance)").Where("`rank` is not null and `rank` > 0").Row().Scan(&totalBalance); err != nil {
```

Comment:
just sum the balance , regardless whether rank is null or not. we should include the balance even vault is not join airdrop

Reply:
Fixed